unicorecmsconnectsmart/templates/home.pt

```diff
@@ -7,42 +7,25 @@
             <div id="banner">
                 <a href="/declaration/">
                     <condition tal:omit-tag="" tal:condition="localisation and localisation.image_host and localisation.image">
-                        <img alt="Welcome to the Girl Effect" src="${view.get_image_url(localisation.image_host, localisation.image, 320)}"/>
+                        <img alt="Welcome to Connect Smart" src="${view.get_image_url(localisation.image_host, localisation.image, 320)}"/>
                     </condition>
                     <condition tal:omit-tag="" tal:condition="not (localisation and localisation.image_host and localisation.image)">
-                        <img alt='Welcome to the Girl Effect' src="/static/img/banner.jpg" />
+                        <img alt='Welcome to Connect Smart' src="/static/img/ConnectSmart_Title-Illustration.jpg" />
                     </condition>
                 </a>
             </div>
         </div>
 
         <div class="what">
-            <div class="h1"><a href="/about/" i18n:translate="What Is The Girl Effect">What Is The Girl Effect</a></div>
+            <div class="h1"><a href="/about/" i18n:translate="What Is Connect Smart?">What Is Connect Smart?</a></div>
             <div class="listing">
-                <a href="/about/"><img alt='What is the Girl Effect' src="/static/img/what_is_girl_effect.png" /></a>
-                <div class="h2"><a href="/about/" i18n:translate="Learn more about The Girl Effect">Learn more about The Girl Effect</a></div>
-                <p i18n:translate="And spread the word">And spread the word</p>
+                <a href="/about/"></a>
+                <div class="h2"><a href="/about/" i18n:translate="Do you have questions about the internet or social media that you're too embarrassed to ask?">Do you have questions about the internet or social media that you're too embarrassed to ask?</a></div>
```

Comment:
It actually adheres to the design with the way the content has been defined, because the article has no description. Do you still want me to change it?
![screen shot 2015-06-25 at 8 52 51 pm](https://cloud.githubusercontent.com/assets/7100966/8362958/3766570a-1b7c-11e5-80bd-98c9d6aae0fe.png)